src/main/c/Makefile

```diff
@@ -11,17 +11,20 @@ endif
 no_cexts: clean
 	@echo "WARNING: TRUFFLERUBY_CEXT_ENABLED is set to false - not building cexts" 1>&2
 
-all: $(ROOT)/lib/cext/ruby.su $(ROOT)/lib/mri/openssl.su $(ROOT)/lib/mri/zlib.su $(ROOT)/lib/mri/syslog.su
+all: $(ROOT)/lib/cext/trufflemock.bc $(ROOT)/lib/cext/ruby.su $(ROOT)/lib/mri/openssl.su $(ROOT)/lib/mri/zlib.su $(ROOT)/lib/mri/syslog.su
```

Comment:
I had to change the order here which hopefully should get the build passing. It was failing to try to link against the non-existing `trufflemock.bc` which didn't exist yet.